projects/demo/src/modules/app/stackblitz/project-files/src/main.ts.md

```diff
@@ -16,6 +16,13 @@ import {App} from './app/app.component';
 class Root {}
 
 bootstrapApplication(Root, {
-  providers: [provideAnimations()],
+  providers: [
+    provideAnimations(),
+    /**
+     * A workaround for StackBlitz only (it does not support assets).
+     * Don't use this approach in real-world applications!
+     */
+    tuiAssetsPathProvider('https://taiga-ui.dev/next/assets/taiga-ui/icons'),
```

Comment:
Gotta remember to remove `next` after we release 4.0 :)

Reply:
I've added `TODO` comment https://github.com/taiga-family/taiga-ui/pull/8156/commits/e394191aef3c8e3853deb1d4a0bbbb68822b6f09